.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: node_js
 
 node_js:
-  - "10"
+  - "11"
 
-script: npm run lint
\ No newline at end of file
+stages:
+  - compile
+  - test
+  - deploy
```

Comment:
It looks like the `stages` configuration is incorrect. The only stages we have defined are `lint` and `unit`, and all of the stages here do not exist.

Additionally, the `stages` key is only really needed to define the order the stages should run for special purposes (such as prepending some random stage). Can we try to remove this block and see if it breaks anything?

Reply:
Removed block successfully